build/mac/cross-compile/run-on-macos.py

```diff
@@ -54,34 +55,44 @@ def read_env_vars(tool):
     src_dir_on_host = require_env_var('MACOS_SRC_DIR_MOUNT')
     keychain_pw = \
         require_env_var('KEYCHAIN_PASSWORD') if requires_keychain(tool) else ''
-    return host, src_dir_on_host, keychain_pw
+    keychain_path = os.getenv('KEYCHAIN_PATH', '')
```

Comment:
is this differ from `--keychain` parameter? can it be just extracted from it?

if you can't parse it from `--keychain`, pls add info on `KEYCHAIN_PATH` into readme.md.

Reply:
The `--keychain` parameter to `codesign` and other tools is optional. (When it isn't given, all keychains are searched.) I therefore believe it is not possible to reliably extract `KEYCHAIN_PATH` from the `--keychain` parameter. I updated the README in 19e0a79725acc6033818a5fe15f392e72f984a80.